src/popup/router/pages/FungibleTokens/TokenDetails.vue

```diff
@@ -0,0 +1,238 @@
+<template>
+  <div class="token-details">
+    <div class="token-header">
+      <div class="token-profile">
+        <Avatar
+          v-if="tokenData.contract || tokenData.image"
+          :address="tokenData.contract"
+          :src="tokenData.image || null"
+          size="xlg"
+        />
+        <div class="amount">
+          <span class="text-ellipsis max-space" :title="tokenData.convertedBalance || '0.00'">{{
+            tokenData.convertedBalance || '0.00'
+          }}</span>
+          <span class="symbol text-ellipsis max-space" :title="tokenData.symbol">{{
+            tokenData.symbol
+          }}</span>
+          <FormatFiatCurrency
+            class="text-ellipsis max-space"
+            :balance="tokenData.balanceCurrency"
+          />
+        </div>
+      </div>
+      <div class="token-actions">
+        <Button bold>
+          <RouterLink :to="{ name: 'send' }">{{ $t('pages.token-details.send') }}</RouterLink>
+        </Button>
+        <Button bold>
+          <RouterLink to="/receive">{{ $t('pages.token-details.receive') }}</RouterLink>
+        </Button>
+        <Button bold :disabled="!tippingSupported">
+          <RouterLink :to="{ name: 'tip' }">{{ $t('pages.token-details.tip') }}</RouterLink>
+        </Button>
+      </div>
+    </div>
+    <TabsMenu v-on:changed="selectTab" :tabOptions="tabs" :activeTab="activeTab" />
+    <div class="token-info">
+      <div class="section-title">
+        {{ $t('pages.token-details.token-details') }}
+      </div>
+      <DetailsRow :label="$t('pages.token-details.symbol')" :text="tokenData.symbol"></DetailsRow>
```

Comment:
```suggestion
      <DetailsRow :label="$t('pages.token-details.symbol')" :text="tokenData.symbol" />
```
do we need an empty text node?